CHANGES

```diff
@@ -1,6 +1,23 @@
 OpenVDB Version History
 =======================
 
+
+Version 8.0.0 - In development
+
+      Some changes in this release (see "ABI changes" below) alter
```

Comment:
Rather verbose and probably unnecessary (the definition of a new major release is an ABI change). How about instead:

This version introduces ABI changes relative to older major releases, so to preserve ABI compatibility it might be necessary to define the macro OPENVDB_ABI_VERSION_NUMBER=N, where, for example, N is 6 for Houdini 18.0 and 7 for Houdini 18.5.